x-pack/plugins/cross_cluster_replication/public/app/sections/home/follower_indices_list/components/context_menu/context_menu.js

```diff
@@ -22,6 +22,8 @@ import {
   FollowerIndexUnfollowProvider
 } from '../../../../../components';
 
+import { getIndexListUri } from '../../../../../../../../index_management/public/services/navigation';
```

Comment:
Is this allowed? 😊 Maybe we should ask @bmcconaghy to expose everything that is _public_ from the `/index_management/index.js` file?

Reply:
Good point! We could even re-export internals from Index Management from a root-level "shared" directory, so we wouldn't need to move any files around. This would allow Index Management's modules to be organized in a way that makes the most sense to Index Management, while still exposing some select functionality to other plugins.

```js
// index_management/public/shared/index.js

export { getIndexListUri } from './services/navigation';
```

Reply:
I will defer this for now as we are doing this deep traversal in other places too, but I think it's a great idea to export from index management root